main.js

```diff
@@ -1 +1,71 @@
 import customers from './customers.js'
+
+
+let person = [{
+    thumbNail: "https://randomuser.me/api/portraits/thumb/women/89.jpg",
+    name: "Pauline Hughes",
+    email: "pauline.hughes@example.com",
+    address: "3747 Elgin St, Roseville, IN 93281"
+}]
+
+
+function capital(string){
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+
+function listCustomers(personArray) {
+    let customerList = document.querySelector('#customer-list');
+    console.log(customerList)
+
+
+    for (let customer of customers) {
```

Comment:
Good work so far. Here you want to loop over `personArray`, which is any array that was passed when the function was called.